extensions/src/services/http-svc.js

```diff
@@ -0,0 +1,32 @@
+
+import axios from 'axios';
+
+/**
+ * This will be the wrapper of actual axios library so we can add
+ * authorization headers for all the extensions.
+ */
+
+let axiosInstance = null;
```

Comment:
Found issues (possible) with such a setup:
* one can't use client if setNexusAuth hasn't been called in advance, because axiosInstance initial value is null
* weird way to use it with getAxios().get

Looks like we can fix this by creating new instance of axios right away with exporting it.
And then when we need to set a token - we alter defaults of that instance to setup headers:
`instance.defaults.headers.common['Authorization'] = AUTH_TOKEN;`